### DISCUSSION TITLE
Information in the wrong place

### DISCUSSION THREAD

**Discussion by ghost** [nokyan/resources#350] [Q&A]
Hello, I would like to report an error, I disable some CPUs and noticed that Resources is putting the clock information in the wrong place, see the image CPU 3 and 4 are in CPU 7 and 8.

![Captura de tela de 2024-08-30 20-27-04](https://github.com/user-attachments/assets/e8d1477e-0ba4-471d-8031-5d63d41bb0fc)

**Comment by nokyan** [comment]
Hi, I'd appreciate if you opened an issue for this, I don't think Discussions are the place for bug reports. :)
But regarding the issue, I'll look into it, though it is quite an edge case.
